src/solver/types.jl

```diff
@@ -115,7 +115,9 @@ struct LargeScaleAdvection{PT, PQ}
     prof_dqtdt::PQ # Set large-scale drying
 end
 # maybe need to <: AbstractForcing
-struct GCMForcing end
+struct GCMForcing{S}
```

Comment:
```suggestion
struct GCMForcing{S <: String}
```
should this subtype string?

Reply:
strings aren't explicitly typed in other structs in `types.jl`, so I'll just follow that convention for now